utilities/checkpoint/checkpoint.cc

```diff
@@ -211,15 +211,16 @@ Status CheckpointImpl::CreateCheckpoint(const std::string& checkpoint_dir) {
     std::vector<std::string> subchildren;
     db_->GetEnv()->GetChildren(full_private_path, &subchildren);
     for (auto& subchild : subchildren) {
-      Status s1 = db_->GetEnv()->DeleteFile(full_private_path + subchild);
+      std::string subchild_path = full_private_path + "/" + subchild;
+      Status s1 = db_->GetEnv()->DeleteFile(subchild_path);
       if (s1.ok()) {
-        Log(db_->GetOptions().info_log, "Deleted %s",
-            (full_private_path + subchild).c_str());
+        Log(db_->GetOptions().info_log, "Delete file %s -- %s",
```

Comment:
fixed now, will always print log when succeed of failed.

On Sun, May 29, 2016 at 4:58 AM, Andrew Kryczka <notifications@github.com>
wrote:

> In utilities/checkpoint/checkpoint.cc
> <https://github.com/facebook/rocksdb/pull/1138#discussion_r64994261>:
>
> >        if (s1.ok()) {
> > -        Log(db_->GetOptions().info_log, "Deleted %s",
> > -            (full_private_path + subchild).c_str());
> > +        Log(db_->GetOptions().info_log, "Delete file %s -- %s",
>
> can you s/Delete/Deleted/ ? I think past tense is clearer for completed
> operations.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/facebook/rocksdb/pull/1138/files/64ec12c253842e01d811a384295b7b14cdfb3368#r64994261>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAgQPXhuGz3uZzU_P08LPbmiCt8PIz6Zks5qGKx-gaJpZM4InPMT>
> .
>